src/Ouzo/Core/Logger/LoggerAdapter.php

```diff
@@ -91,6 +91,56 @@ public function log(string $level, $message, array $context = []): void
         }
     }
 
+    public function asLoggerInterface(): LoggerInterface
+    {
+        return new class() implements LoggerInterface {
+            public function log($level, $message, array $context = []): void
```

Comment:
Type `string` for `$level` is missing. All other methods too.

Reply:
I can't set type this because if I do it, the signature is incompatible with LoggerInterface. Maybe we need use https://github.com/php-fig/log/tree/master instead of  "psr/log"?